stackit-postgres-db/tests/postgres.tftest.hcl

```diff
@@ -1,44 +1,45 @@
 mock_provider "stackit" {
   mock_resource "stackit_postgresflex_instance" {
     defaults = {
-      instance_id = "87778dd7-a506-45e8-9cd8-134585230603"
+      instance_id = "aeac146a-97d6-4677-91eb-6ab5f8b0c202"
     }
   }
-
   mock_resource "stackit_postgresflex_user" {
     defaults = {
       password = "password"
-      host     = "https://example.com"
+      host     = "postgres.stackit.internal"
     }
   }
-
-  mock_resource "stackit_postgresflex_database" {
-  }
+  mock_resource "stackit_postgresflex_database" {}
 }
 
 mock_provider "vault" {
-  mock_resource "vault_kv_secret_v2" {
-  }
+  mock_resource "vault_kv_secret_v2" {}
+}
+
+variables {
+  project_id = "aeac146a-97d6-4677-91eb-6ab5f8b0c202"
 }
 
-# Test 1: Basic instance creation with required variables
-run "basic_instance_creation" {
+# --- Test 1: Basic Creation ---
+run "basic_creation" {
   command = apply
 
   variables {
-    name                 = "test-postgres"
-    project_id           = "aeac146a-97d6-4677-91eb-6ab5f8b0c202"
-    cpu                  = 2
-    memory               = 4
-    engine_version       = "17"
-    disk_size            = 10
-    acls                 = ["10.0.0.0/16"]
-    manage_user_password = false
-  }
+    name           = "test-postgres"
+    project_id     = var.project_id
+    cpu            = 2
+    memory         = 4
+    engine_version = "17"
+    disk_size      = 10
```

Comment:
This can probably also be set in the file `variables` block to not repeat it in every `run`